hoot-core-test/src/test/cpp/hoot/core/io/OgrReaderTest.cpp

```diff
@@ -84,7 +84,7 @@ class OgrReaderTest : public CppUnit::TestFixture
 
       Coordinate c1(-1, 84);
       Coordinate c2(1, 84);
-      double d = GeometryUtils::haversine(c1, c2);
+      /*double d =*/ GeometryUtils::haversine(c1, c2);
```

Comment:
There's not.  That was an unrelated commit to get rid of a compiler warning...will remove.